core.md

```diff
@@ -0,0 +1,111 @@
+
+A member may choose to become inactive and appoint another in their place until they return.
+
+At least one member of the Council is promoted from other teams via an annual internal vote, to ensure that the Council always has a fresh opinion and does not stagnate over time.
+
+Once a year, [an internal voting process](https://peps.python.org/pep-8104/) will determine the members that will be promoted to the Steering Council, via the ranked voting system. It is intended that at least one, and at most three members will be promoted at a time.
+
+The Council also serves double duty as a mediator for disagreements between other teams. These shall be resolved by discussion, rather than by unanimous applications of the rules, so this is not in the purview of this document.
+
+#### Maintainers
+
+The Maintainers team consists of all project developers who have push access or technical rights to the project's repositories or infrastructure.
+
+They are the ones keeping the Project's repositories up to date, and ensuring the quality and standards of the Project are upheld, as well as adding new features as the community comes to require them.
+
+Though maintainers may individually or collectively have different access rights across the project's many repositories or infrastructure services, all maintainers within the team are considered equal to each other with respect to their rights within the team's governance.
+
+Maintainers may be appointed by other maintainers, or by application from the person who wishes to join the team.
+
+
+#### Moderators
+
+The Moderation team is composed of all moderators of all community spaces (GitHub, Discord) under the purview of the project.
+
+They have the responsibility of enforcing the policies and rules of each space, as well as the Code of Conduct, and applying disciplinary sanctions to people who violate said rules.
+
+#### Subproject Leads
+
+A special group exists for those designated as "subproject leaders".
+
+The members of this group were assigned from the Maintainers team, and is indeed a strict subset of the Maintainers team.
+
+The role of this team is to take lead of a subproject, as defined above.
+
+The subproject leader has total purview of their given subproject (repository or task) up to and including their primary goal.
+
+They may:
+* Decide the structure of their team, in whatever capacity is neccessary
+* Decide who may or may not be on their team
+* Delegate the power of subproject leader to another person
+* Make final decisions on the implementation and structure of their subproject
+* Overrule another Maintainer on the strict matter of their subproject, and no other.
+
+They are not:
+* A project dictator
+* Unaccountable
+* Free from scrutiny
+
+They may not:
+* Make decisions not related to their subproject
+* Make major architectural decisions without the assent of the Steering Council
```

Comment:
Why the Council directly and not the team assigned to the project first?

Reply:
I specifically meant major architectural decisions about the overall Project / framework, outside of the exclusive jurisdiction of the subproject.